### PR TITLE
fix(autotracing): honor dload sampling interval

### PR DESCRIPTION
**Pull request by zjncs**
## Summary

Make dload event tracing use its configured sampling interval.

## Problem

`newDload` validated and stored `Config.Dload.Interval` in the tracer, but returned an `EventTracingAttr` with a hard-coded 30-second interval. As a result, changing the documented sampling interval had no effect on scheduler cadence.

## Changes

- Derive the registered event interval from the validated tracer interval.
- Add a regression test configuring 7 seconds and asserting the event attribute reports 7.

## Verification

- Base SHA: `$(git rev-parse main)`; current main inspected before change.
- `git diff --check` passed.
- Go tests could not be executed because the workspace environment does not provide `go`/`gofmt`; please let CI validate the Go test.

## Risk

Low; this restores the configured behavior and leaves default configuration unchanged.

AI-assisted contribution; implementation and verification performed against current main.

### CODE REVIEW

File: core/autotracing/dload_test.go [hunk 1]
```diff
@@ -224,3 +224,17 @@ func TestUpdateLoad(t *testing.T) {
 		t.Fatalf("loadAvg = %v, want %v", info.loadAvg, expectedLoadAvg)
 	}
 }
+
```
Comment (huhong-web) [inline core/autotracing/dload_test.go:227]: [P2] [CR-001] The test mutates global config without restoring it, so later package tests can become order-dependent. Snapshot the old config and restore it with t.Cleanup.\n\nFix: add the focused regression described above.

**Review by huhong-web — COMMENTED** [review]
## Code Review

One non-blocking finding is included inline: core/autotracing/dload_test.go:227.

## Architecture & Design
- Verdict: Reasonable with a focused verification improvement.
- Evidence: core/autotracing/dload_test.go:227 changes a behavioral contract without proving the stated edge case.

## Scope & Focus
- Verdict: Focused
- Primary objective: Honor the configured sampling interval.
- Necessary changes: core/autotracing/dload_test.go:227 and a focused regression test.
- Out-of-scope changes: None.
- Split plan: Not required.

## Review Convergence
- Mode: full
- Base SHA: ae0badc0a1fed0a1403ff352edd33723aeafaf3e
- Reviewed HEAD: c8cc8dd66746fa604812fc6c1e0123d51832933c
- Previous reviewed HEAD: none
- Scope verdict: Focused
- Coverage: Changed files and direct configuration/test contracts; two independent review perspectives.
- Prior findings: 0 open / 0 fixed / 0 verified / 0 withdrawn
- New findings: 1 FIRST_PASS / NON-BLOCKING
- Late non-blocking findings: None
- Full-review reset reason: Not required
- Completeness declaration: At HEAD c8cc8dd66746fa604812fc6c1e0123d51832933c, all changed human-authored files and assigned dimensions were covered; all known blockers are included.

## Verification
- Build/Lint/Integration: NOT RUN — prohibited by review no-build policy.
- Targeted verification: Not required.

---
*Co-authored by: gpt-5.6-terra*
